plrust/Cargo.toml

```diff
@@ -60,6 +60,7 @@ prettyplease = "0.1"
 syn = "1"
 quote = "1"
 proc-macro2 = "1"
+omnipath = "0.1.1"
```

Comment:
I glanced at this and the implementation seems POSIXly correct, yeah, and the library is sound (and also more annoying to maintain ourselves because it's one of those extremely pedantic things by nature).

Reply:
It's actually wrong to use in plrustc, I think, since we should only ever allow canonicalized paths. Annoying.

Reply:
Now only plrust (not plrustc) uses it, where it seems fine.